definitions/npm/ansi-regex_v4.x.x/flow_v0.25.x-/ansi-regex_v4.x.x.js

```diff
@@ -0,0 +1,10 @@
+declare module 'ansi-regex' {
+  declare export type Options = {
+    /**
+     * Match only the first ANSI escape.
+     */
+    onlyFirst?: boolean,
+  };
+  declare function ansiRegex(options?: Options): RegExp;
+  declare module.exports: typeof ansiRegex;
```

Comment:
You could probably also declare this like `declare export default function ansiRegex(options?: Options): RegExp;`